frontend/src/layout/scenes/SceneHeader.tsx

```diff
@@ -0,0 +1,159 @@
+                            <LemonButton
+                                onClick={() => setScenePanelOpen(!scenePanelOpen)}
+                                icon={<IconInfo className="text-primary" />}
+                                tooltip={scenePanelOpen ? 'Close info panel' : 'Open info panel'}
+                                active={scenePanelOpen}
+                                size="small"
+                            />
+                        )}
+
+                        <TopBarSettingsButton buttonProps={{ size: 'small', icon: <IconGear /> }} />
+                    </div>
+                </div>
+            </div>
+        </>
+    ) : null
+}
+
+interface BreadcrumbProps {
+    breadcrumb: IBreadcrumb
+    here?: boolean
+    isOnboarding?: boolean
+}
+
+function Breadcrumb({ breadcrumb, here, isOnboarding }: BreadcrumbProps): JSX.Element {
+    const { assureVisibility } = useActions(projectTreeLogic({ key: PROJECT_TREE_KEY }))
+    const { showLayoutPanel, setActivePanelIdentifier } = useActions(panelLayoutLogic)
+    const [popoverShown, setPopoverShown] = useState(false)
+
+    const joinedKey = joinBreadcrumbKey(breadcrumb.key)
+
+    const breadcrumbName = isOnboarding && here ? 'Onboarding' : (breadcrumb.name as string)
+
+    let nameElement: JSX.Element
+    if (breadcrumb.symbol) {
+        nameElement = breadcrumb.symbol
+    } else {
+        nameElement = (
+            <span className={cn('flex items-center gap-1.5 inline-block whitespace-nowrap')}>
+                {breadcrumbName === '' ? <em>Unnamed</em> : breadcrumbName}
+                {'tag' in breadcrumb && breadcrumb.tag && <LemonTag size="small">{breadcrumb.tag}</LemonTag>}
+            </span>
+        )
+    }
+
+    const isProjectTreeFolder = Boolean(
+        breadcrumb.name && breadcrumb.path && 'type' in breadcrumb && breadcrumb.type === 'folder'
+    )
+
+    const Component = !isProjectTreeFolder && breadcrumb.path ? Link : 'span'
+    const breadcrumbContent = (
+        <Component
+            onClick={() => {
+                breadcrumb.popover && setPopoverShown(!popoverShown)
+                if (isProjectTreeFolder && breadcrumb.path) {
+                    assureVisibility({ type: 'folder', ref: breadcrumb.path })
+                    showLayoutPanel(true)
+                    setActivePanelIdentifier('Project')
+                }
+            }}
+            data-attr={`breadcrumb-${joinedKey}`}
+            to={breadcrumb.path}
+            className={cn('text-primary text-sm inline-grid', {
+                'font-bold': here,
+            })}
+        >
+            {nameElement}
+            {breadcrumb.popover && !breadcrumb.symbol && <IconChevronDown />}
+        </Component>
+    )
+
+    if (breadcrumb.isPopoverProject) {
+        return (
+            <ProjectDropdownMenu
+                buttonProps={{
+                    size: 'xxs',
+                    className: 'text-primary font-normal p-0 hover:text-primary gap-1',
```

Comment:
I think the discrepancy between https://github.com/PostHog/posthog/pull/35033/files#diff-21d22d88766cebb0032089897ced4ffa7db545215ab1b842ea0c02a4f8617be7R258 and this line is causing the issues you're seeing locally when the flag isn't enabled